src/content-helper/dashboard-page/pages/traffic-boost/preview/hooks/use-iframe-highlight.ts

```
@@ -0,0 +1,597 @@
+
+		let node;
+		let fullText = '';
+		const nodePositions: {
+			node: Text;
+			start: number;
+			end: number;
+			blockParent: Element | null;
+		}[] = [];
+
+		// Build full text and track node positions.
+		while ( ( node = treeWalker.nextNode() as Text ) ) {
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)